base/applications/rapps/appview.cpp

```diff
@@ -141,7 +141,7 @@ HWND CMainToolbar::Create(HWND hwndParent)
         return FALSE;
     }
 
-    ImageList_Destroy(SetImageList(hImageList));
+    SetImageList(hImageList);
```

Comment:
This destroys the previous imagelist.

Reply:
Yes but TOOLBAR_Destroy will later try to destroy the same Imagelist, as set as component Internal Imagelist by SetImageList (inherited from CToolbar).
Same problem as for https://github.com/reactos/reactos/pull/3211 (where the issue came from the use of a ListView)

Reply:
Please explain how TOOLBAR_Destroy will delete the imagelist that is being leaked here.
Does it keep a list of previously returned imagelists?

Reply:
- hImageList get referenced as himlInt in the Toolbar component when SetImageList is called
- then destroyed L144 by ImageList_Destroy
- and himlInt gets deleted again when TOOLBAR_Destroy is called (when component is destroyed at RAPPS closure)

Step 2 is a bug.